HIP/hip-xxx.md

```diff
@@ -0,0 +1,921 @@
+                },
+                "reason": null
+            },
+            {
+                "pc": 543,
+                "op": "POP",
+                "gas": 4416,
+                "gasCost": 2,
+                "depth": 1,
+                "stack": null,
+                "memory": [
+                    "0x0000000000000000000000000000000000000000000000000000000000000000",
+                    "0000000000000000000000000000000000000000000000000000000000000000",
+                    "0000000000000000000000000000000000000000000000000000000000000080"
+                ],
+                "storage": {
+                    "0x0000000000000000000000000000000000000000000000000000000000000000": "0x0000000000000000000000000000000000000000000000000000000000000014"
+                },
+                "reason": null
+            },
+            {
+                "pc": 544,
+                "op": "PUSH1",
+                "gas": 4414,
+                "gasCost": 3,
+                "depth": 1,
+                "stack": null,
+                "memory": [
+                    "0x0000000000000000000000000000000000000000000000000000000000000000",
+                    "0000000000000000000000000000000000000000000000000000000000000000",
+                    "0000000000000000000000000000000000000000000000000000000000000080"
+                ],
+                "storage": {
+                    "0x0000000000000000000000000000000000000000000000000000000000000000": "0x0000000000000000000000000000000000000000000000000000000000000014"
+                },
+                "reason": null
+            },
+            {
+                "pc": 546,
+                "op": "SLOAD",
+                "gas": 4411,
+                "gasCost": 2100,
+                "depth": 1,
+                "stack": null,
+                "memory": [
+                    "0x0000000000000000000000000000000000000000000000000000000000000000",
+                    "0000000000000000000000000000000000000000000000000000000000000000",
+                    "0000000000000000000000000000000000000000000000000000000000000080"
+                ],
+                "storage": {
+                    "0x0000000000000000000000000000000000000000000000000000000000000000": "0x0000000000000000000000000000000000000000000000000000000000000014"
+                },
+                "reason": null
+            },
+            [...]
+        ]
+}
+```
+
+For providing output formatted by `opcodeLogger`, the Mirror node will re-execute the transaction using the state from the `contract_state_changes` sidecars produced by the consensus nodes. In this way, we can have a track on all of the storage/memory information and the entire trace of opcodes that were executed during the replay.
+
+⚠️ The output generated from the opcodeLogger is very verbose. The /traceOpcodes endpoint will be disabled by default and can be enabled by users of the Hedera Local node or users running their own Mirror Nodes.
```

Comment:
I believe `/traceOpcodes` should be changed to `/opcodes`

Reply:
Fixed